monix-eval/shared/src/test/scala/monix/eval/TaskFromFutureSuite.scala

```diff
@@ -88,264 +88,253 @@ object TaskFromFutureSuite extends BaseTestSuite {
     assertEquals(result.value, Some(Success(1)))
   }
 
-  test("Task.fromFuture(allowContinueOnCallingThread = true) should be faster for completed futures, success") {
-    implicit s =>
-      val t = Task.fromFuture(Future.successful(10), allowContinueOnCallingThread = true)
-      val f = t.runToFuture
-      assertEquals(f.value, Some(Success(10)))
+  test("Task.fromFuture() should be faster for completed futures, success") { implicit s =>
```

Comment:
We could remove some of the tests here too and just keep those that check for faster completion

Perhaps it will be faster to just revert to the state from before my changes